packages/odata-v2/src/de-serializers/payload-value-converter.ts

```diff
@@ -1,6 +1,13 @@
 import { EdmTypeShared } from '@sap-cloud-sdk/odata-common/internal';
 import { DeSerializers } from './de-serializers';
 
+/**
+ * Convert a value with edm format to one with typescript format. This function is typically used when deserializing an OData response.
+ * @param value - The original value with edm format.
+ * @param edmType - The edm type of that value.
```

Comment:
[req] Please use the correct casing in docs: "EDM" and "TypeScript".
```suggestion
 * Convert a value from EDM format to TypeScript. This function is typically used when deserializing an OData response.
 * @param value - The original value with EDM format.
 * @param edmType - The EDM type of that value.
```